homeassistant/components/upnp/device.py

```diff
@@ -140,30 +164,32 @@ def model_name(self):
         except (asyncio.TimeoutError, aiohttp.ClientError, UpnpError):
             _LOGGER.error("Could not delete port mapping")
 
-    async def async_get_total_bytes_received(self):
-        """Get total bytes received."""
-        try:
-            return await self._igd_device.async_get_total_bytes_received()
-        except asyncio.TimeoutError:
-            _LOGGER.warning("Timeout during get_total_bytes_received")
-
-    async def async_get_total_bytes_sent(self):
-        """Get total bytes sent."""
-        try:
-            return await self._igd_device.async_get_total_bytes_sent()
-        except asyncio.TimeoutError:
-            _LOGGER.warning("Timeout during get_total_bytes_sent")
-
-    async def async_get_total_packets_received(self):
-        """Get total packets received."""
-        try:
-            return await self._igd_device.async_get_total_packets_received()
-        except asyncio.TimeoutError:
-            _LOGGER.warning("Timeout during get_total_packets_received")
+    async def async_get_traffic_data(self) -> Mapping[str, Mapping[str, any]]:
```

Comment:
Looks like the return value typing should be adjusted.
```suggestion
    async def async_get_traffic_data(self) -> Mapping[str, Union[str, any]]:
```

Reply:
Should just be `Mapping[str, any]`